win/rl/winrl.cc

```diff
@@ -554,9 +560,10 @@ NetHackRL::status_update_method(int fldidx, genericptr_t ptr, int,
         return;
 
     // Needs to be kept in sync with the switch statement in rl_status_update.
-    if (fldidx == BL_FLUSH || fldidx == BL_RESET)
+    if (fldidx == BL_FLUSH || fldidx == BL_RESET) {
```

Comment:
For context on the logic here, see e.g. https://github.com/NetHack/NetHack/blob/b65c93cdff3f52483645b29fc68f4adeb39ccbf3/doc/window.doc#L422